ls1intum/Artemis
# `General`: Add placeholder message if student isn't enrolled in any courses

### Checklist
#### General




#### Client


### Motivation and Context


Resolves #8325

The dashboard seems rather empty without a placeholder if the student isn't enrolled in any courses. Adding a small placeholder improves the UI.

### Description
Added a text placeholder to the dashboard if the student isn't enrolled in any courses.


### Steps for Testing

Prerequisites:
- 1 Student
- 1+ Courses

1. Log in to Artemis as Student
2. Check if placeholder is being shown
3. Enroll manually in any course
4. Check if placeholder disappears

### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage







### Screenshots


![Screenshot 2024-04-29 160722](https://github.com/ls1intum/Artemis/assets/98264403/452cf6fb-bb99-4cb8-a557-eb92a22404c4)
